Xamarin.Forms.Core/Touch.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+
+namespace Xamarin.Forms
+{
+	public class Touch
+	{
+		public Touch(int touchIndex, TouchPoint touchPoint, View view)
```

Comment:
How does it work for multiple touches ?

Reply:
Each touch has its own TouchId/touchIndex